apps/rod/views/content/proof-identity.md

```diff
@@ -0,0 +1,11 @@
+# For proof of identity only
```

Comment:
I would suggest to add header in pages.json to make the <title> render from layout.html.